src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -636,11 +636,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                             ' Use the operator methods. Figure out the precise rules first.
                             Return Nothing
                         Else
+                            ' Operators are not used here.  So this is all simple, non-lifted stepping.
```

Comment:
The case where there are user defined operators is in the higher 'if' branch (which is currently not implemented for ioperation). That was my reading of the code at least.